.travis.yml

```diff
@@ -11,14 +12,20 @@ before-script:
 script:
   - docker run my-test-image tests/test_viewmodel.py
   - docker run my-test-image tests/test_integration.py
-  - >
-     docker run
-     -e COSMOS_MONGO_DB_CONNECTION=$COSMOS_MONGO_DB_CONNECTION
-     -e COSMOS_MONGO_DB_DATABASE=$COSMOS_MONGO_DB_DATABASE
-     my-test-image tests/test_e2e.py
+ # - >
+  #   docker run
+  #   -e COSMOS_MONGO_DB_CONNECTION=$COSMOS_MONGO_DB_CONNECTION
+  #   -e COSMOS_MONGO_DB_DATABASE=$COSMOS_MONGO_DB_DATABASE
+  #   my-test-image tests/test_e2e.py
```

Comment:
Did your e2e test stop working? Do you just need to update Travis with the correct connection string?